docs/tool/js/views/map-view.js

```diff
@@ -135,9 +135,23 @@
                 .domain([MIN_DOMAIN_VALUE, MAX_DOMAIN_VALUE])
                 .range([MIN_COLOR, MAX_COLOR]);
 
+            var roundedVersionOf = function(val){
+                switch(style){
+                    case "percent":
+                        return Math.roundTo(val, .01);
+                    case "money":
+                        return Math.roundTo(val, 100);
+                    case "number":
+                        return Math.roundTo(val, 100);
+                    default:
+                        return Math.round(val);
+                }
+            }
+
             this.stops = new Array(CHLOROPLETH_STOP_COUNT).fill(" ").map(function(el, i){
                 var stopIncrement = MAX_DOMAIN_VALUE/(CHLOROPLETH_STOP_COUNT - 1);
-                var domainValue = Math.round((MAX_DOMAIN_VALUE - (stopIncrement * i))*1000)/1000; //stupid javascript rounding. Close enough for this purpose. 
+                var domainValue = MAX_DOMAIN_VALUE - (stopIncrement * i);
+                var domainValue = roundedVersionOf(domainValue);
```

Comment:
do we want the second `var` here? Does it even make a difference to re-var the variable?